### PR TITLE
fix(WidgetRepresentation): Use device pixel ratio

### PR DESCRIPTION
**Pull request by laurennlam**
### Changes

- [ ] Documentation and TypeScript definitions were updated to match those changes

### PR and Code Checklist

- [ ] [semantic-release](https://github.com/semantic-release/semantic-release) commit messages
- [ ] Run `npm run reformat` to have correctly formatted code

### Testing

- [ ] This change adds or fixes unit tests 
- [ ] Tested environment:
  - **vtk.js**: 
  - **OS**: 
  - **Browser**:

### CODE REVIEW

**Comment by laurennlam** [comment]
@finetjul can you test it please ?

File: Sources/Widgets/Representations/WidgetRepresentation/index.js [hunk 1]
```diff
@@ -221,7 +221,7 @@ function vtkWidgetRepresentation(publicAPI, model) {
     }
 
     const rHeight = rendererPixelDims[1];
-    return scale / rHeight;
+    return (window.devicePixelRatio * scale) / rHeight;
```
Comment (finetjul) [inline Sources/Widgets/Representations/WidgetRepresentation/index.js:224]: are parenthesis needed ?

**Comment by finetjul** [comment]
This does not seem right, it would be a regression from #2030 

I'll have a deeper look

File: Sources/Widgets/Representations/WidgetRepresentation/index.js (hunk 1 above)
Comment (jourdain) [reply Sources/Widgets/Representations/WidgetRepresentation/index.js:224]: Also that assume the vtkRenderWindow use `window.devicePixelRatio` to scale up its resolution.

**Pull request closed without merging**